src/Jackett/JackettModule.cs

```diff
@@ -59,6 +59,10 @@ protected override void Load(ContainerBuilder builder)
             Mapper.CreateMap<WebClientByteResult, WebClientStringResult>().ForMember(x => x.Content, opt => opt.Ignore()).AfterMap((be, str) =>
             {
                 str.Content = Encoding.UTF8.GetString(be.Content);
+                if(str.Content.Contains("NorBits"))

```

Comment:
Mmmh i'm not very happy to see this in this core file.
@flightlevel do you have an idea on how we can do it différent ?
Maybe by recquiring Encoding on provider side and assuming that if this setting is not populated (for older providers),
We assume that content is UTF8